apps/app_queue.c

```diff
@@ -8871,11 +8876,33 @@ static int queue_exec(struct ast_channel *chan, const char *data)
 
 	cid_allow = qe.parent->log_restricted_caller_id || ((ast_party_id_presentation(&ast_channel_caller(chan)->id) & AST_PRES_RESTRICTION) == AST_PRES_ALLOWED);
 	
-	ast_queue_log(args.queuename, ast_channel_uniqueid(chan), "NONE", "ENTERQUEUE", "%s|%s|%d",
-		S_OR(args.url, ""),
-		S_COR(cid_allow && ast_channel_caller(chan)->id.number.valid, ast_channel_caller(chan)->id.number.str, ""),
-		qe.opos);
-
+	if (log_cidname) {
+	   char *escaped_cidname = NULL;
+
+    /* Ensure caller ID name is valid and not NULL before processing */
```

Comment:
I'm afraid the indentation here still appears to be incorrect. It looks like all your changes are using spaces instead of tabs, in some case there is no actual indentation.